webviz_subsurface/plugins/_co2_leakage/views/mainview/settings.py

```diff
@@ -361,6 +419,18 @@ def __init__(
                             ],
                             style=self._CM_RANGE,
                         ),
+                        "Visualization threshold",
+                        html.Div(
+                            [
+                                dcc.Input(id=visualization_threshold_id,type="number"),
+                                dcc.Checklist(
+                                    ["Show 0"],
+                                    ["Show 0"],
```

Comment:
Ev kunne disse ligget et annet sted, ala som for "options" i noen av de andre over. Gjleder i så fall også "Auto". Men ikke så farlig